src/parcsr_ls/par_relax.c

```diff
@@ -762,6 +762,8 @@ hypre_BoomerAMGRelaxHybridGaussSeidel_core( hypre_ParCSRMatrix *A,
       v_buf_data = (HYPRE_Real *) hypre_ParCSRCommHandleSendDataBuffer(persistent_comm_handle);
       v_ext_data  = (HYPRE_Real *) hypre_ParCSRCommHandleRecvDataBuffer(persistent_comm_handle);
 #else
+      HYPRE_Int num_cols_offd = hypre_CSRMatrixNumCols(A_offd);
```

Comment:
Why defining again? Is it different from the one at line 714?

Reply:
Agreed! Let me remove this redundant line

Reply:
Fixed with 6df6f9b